audit_log/models/managers.py

```diff
@@ -8,6 +8,7 @@
 from django.conf import settings
 
 from audit_log.models.fields import LastUserField
+from audit_log import settings
```

Comment:
This clashes with the "from django.conf import settings" line above, and breaks retrieving the settings.AUTH_USER_MODEL attribute.